docs/colang_2/overview.rst

```diff
@@ -89,9 +89,9 @@ You can migrate your Colang 2.0-alpha bots to 2.0-beta using the following comma
 .. code-block:: console
 
     nemoguardrails convert "path/to/2.0-alpha/version/bots" --from-version "2.0-alpha"
-Additionally, you can enable the `--validate` flag to check if the migrated files do not raise any Colang syntax errors:
-.. code-block: console
```

Comment:
why remove the example?

Reply:
What is the downside of using --validate ? Why not always use it (e.g. just show one proposed way to do it and explaing that users can omit the --validate flag if they don't want to be informed about Syntax errors.